R/utilities-data-combined.R

```diff
@@ -214,12 +214,18 @@ calculateResiduals <- function(dataCombined,
       observedData$xValues
     )$y
   )
-
   # Residual computation will depend on the scaling.
   if (scaling %in% c(tlf::Scaling$lin, tlf::Scaling$identity)) {
     pairedData <- dplyr::mutate(pairedData, residualValues = yValuesSimulated - yValuesObserved)
   } else {
-    pairedData <- dplyr::mutate(pairedData, residualValues = log(yValuesSimulated) - log(yValuesObserved))
+    # Epsilon for safe log calculation should be converted to the units of the values
+    epsilon <- toUnit(
```

Comment:
The epsilon should only be valid for values in base unit. So have to convert it here to the actually used unit.

Reply:
hum..good point